tf2onnx/graph.py

```diff
@@ -656,16 +670,25 @@ def reset_nodes(self, ops):
         self.contained_graphs = remained_sub_graphs
         self._nodes_by_name = {op.name: op for op in ops}
         self._output_to_node_name = {}
+        self._output_to_consumers = {}
         for op in ops:
             for op_output in op.output:
                 self._output_to_node_name[op_output] = op.name
+            inps = op.input
+            for op_input in inps:
+                self._register_input_name(op_input, op)
 
         for n in self._order_sensitive_inputs:
             if n not in ops:
                 self._order_sensitive_inputs.remove(n)
         for o in self.outputs:
             if o not in self._output_to_node_name:
                 raise ValueError("graph output " + o + " not exist")
+        for i in self.inputs:
+            if i.is_graph_input():
+                continue
+            if i.name not in self._output_to_consumers:
+                raise ValueError("graph input %r not exist in graph." % i.name)
```

Comment:
```suggestion
```

Reply:
Right. I did not check inputs was a property and was returning inputs verifying this condition.  I removed the line.